README.md

```diff
@@ -89,3 +89,25 @@ $ git tag -d v1.2.3                 # delete tag v1.2.3 locally
 $ git push origin v1.2.3            # push tag v1.2.3
 $ git push -d origin v1.2.3         # delete tag v1.2.3 remotely
 ```
+
+## Design patterns
+
+#### JS concepts
+
+- The repo is set up to import individual components, which then in turn import their own scss modules. 
+- Preference for hooks and functional components over classes.
+- Preference for proptypes validation over typescript and .d.ts files. 
+- Please try to avoid using `<Media>` and `<Spacer>` components for now, they will be deprecated soon. For media queries, import `@include` rules from the media scss file as described below.
+
+#### CSS/SCSS concepts
+
+- CSS variables are the core of styling in the EDS. 
+  - All CSS variables are available on all pages in both monorepo and cms; they are the single source of truth for brand colors, breakpoints, and other commonly reused bits of css.
+- The EDS is (to an extent) designed as a set of immutable components in terms of interior JS logic as well as styling. New styling of existing components should be handled by PR's on the EDS, not by writing local overrides in files. Due to the generated hashed classnames from scss modules, it is almost impossible to sidestep styling rules of the EDS repo from monorepo or cms.
+- We encourage a limited subset of SCSS features for the EDS.
+  - The primary reason for using scss is for the [nesting syntax](https://sass-lang.com/guide#topic-3). 
+  - The secondary reason for using scss is to import @include statements from the media scss file for reusable media queries, like this:
```

Comment:
Not sure these are the primary and secondary reasons per se. @zach-ethos do you agree with these assertions?

In terms of the primary reason, I personally hate sass's ease of nesting and it causes all sorts of overly specific resulting CSS code.

Secondary reason seems to make sense.

I don't see the value of making the last statements regarding using `calc`, but we can leave it in here for now if you feel strongly. Ultimately, all of this stuff will get ported into some sort of Guidelines area of the EDS https://app.asana.com/0/1137984670709817/1137983928373132/f